cmd/clusterctl/examples/openstack/ubuntu/provider-components.yaml.template

```diff
@@ -249,6 +249,17 @@ data:
         EOF
         systemctl daemon-reload
         systemctl restart kubelet.service
+        
+        # Reset iptables config  
+        cat > /etc/systemd/system/docker.service.d/iptables.conf <<EOF
+        [Service]
+        # Yes, we need to clear it, because otherwise it is addative!
+        ExecStart=
+        ExecStart=/usr/bin/dockerd -H fd:// --iptables=false --ip-masq=false
+        EOF
+        systemctl daemon-reload
+        systemctl disable ufw 
```

Comment:
Please add some whitespace above and a comment referencing this PR/Issue  as to why we disable `ufw`

Reply:
We don't have access from inside the pod because ufw block the traffic.

We need to allow external clients such as kubectl and kubelet to access the Kubernets API, so we should disable the firewall or create new iptables rules.